crates/uv/src/commands/venv.rs

```diff
@@ -162,8 +162,16 @@ async fn venv_impl(
             Err(WorkspaceError::MissingProject(_)) => None,
             Err(WorkspaceError::MissingPyprojectToml) => None,
             Err(WorkspaceError::NonWorkspace(_)) => None,
+            Err(WorkspaceError::Toml(path, err)) => {
+                warn_user!(
+                    "Failed to parse `{}` during environment creation:\n{}",
+                    path.user_display().cyan(),
+                    textwrap::indent(&err.to_string(), "  ")
+                );
```

Comment:
This matches the general style and structure of the comparable error we show in settings discovery:

https://github.com/astral-sh/uv/blob/b38d3fec64c03f299cf203e4a7dd9ababdb3c377/crates/uv-settings/src/lib.rs#L82